spec/rails_helper.rb

```diff
@@ -37,10 +37,6 @@
 end
 
 RSpec.configure do |config|
-  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
```

Comment:
fixture_path is deprecated now; I use FactoryBot, so all I had to do was to delete the corresponding config.